### ISSUE TITLE
Add a prompt to install the missing jq dependency

### ISSUE THREAD

**Issue by stepdg** [mag37/dockcheck#110]
Please add a prompt to install jq if it is missing the same as you do if regctl is missing.  Great work as always!

**Comment by mag37** [comment]
Good suggestion! 

It's in all the major **packagemanagers** though, so people should preferably get it that way instead of grabbing a static binary that wont be updated. But I'll consider the alternatives.
- Only trigger a y/n and grab the static bin as with regctl.
- Add a text stating the suggested way is with **packagemanager**, but optionally download static bin y/n.
- Build a function to check/guess what distribution (and thereby packagemanager) is used, else static bin.

I'll do some thinking, but I think you're right it should be included somehow.

**Comment by mag37** [comment]
I thought it was a good idea and decided to rewrite some current dependency download -logic. 
There's a new branch [dependency_download_rewrite](https://github.com/mag37/dockcheck/tree/dependency_download_rewrite) - I'm still doing some testing before pushing to main.

It will prompt for install, then check packagemanager and if that fails it'll fall back to static binary download.
Test it if you'd like!

**Comment by yoyoma2** [comment]
The change notes are not working in 0.5.0. Not sure it's worth logging an issue. Maybe a note in the 0.5.1 release notes as it may already work in 0.5.1 when 0.5.2 eventually comes out.
```
New version available! v0.5.0 ⇒ v0.5.1
 Change Notes:
[##################################################] 2/2
```

**Comment by yoyoma2** [comment]
> Hotfixed it v0.5.0->v0.5.1

Confirmed fixed! Didn't upgrade to 0.5.1 yet but the change notes appear now.

**Comment by stepdg** [comment]
0.5.1 worked flawlessly

**Comment by mag37** [comment]
@yoyoma2 

> Confirmed fixed! Didn't upgrade to 0.5.1 yet but the change notes appear now.

This will still be an issue going from 0.5.0 to 0.5.2 I'm afraid, but from 0.5.1 onwards it should be fixed.
